Source/Other/SSHTunnel/SPSSHTunnel.m

```diff
@@ -396,10 +393,8 @@ - (void)launchTask:(id) dummy
 			// Set a custom control path to isolate connection sharing to Sequel Ace, to prevent picking up
 			// existing masters without forwarding enabled and to isolate from interactive sessions.  Use a short
 			// hashed path to aid length limit issues.
-			unsigned char hashedPathResult[16];
 			NSString *pathString = [NSString stringWithFormat:@"%@@%@:%ld", sshLogin?sshLogin:@"", sshHost, (long)(sshPort?sshPort:0)];
-			CC_MD5([pathString UTF8String], (unsigned int)strlen([pathString UTF8String]), hashedPathResult);
-			NSString *hashedString = [[[NSData dataWithBytes:hashedPathResult length:16] dataToHexString] substringToIndex:8];
+			NSString *hashedString = [pathString.sha256Hash substringToIndex:8];
 			TA(@"-o",([NSString stringWithFormat:@"ControlPath=%@/SPSSH-%@", [NSFileManager temporaryDirectory], hashedString]));
```

Comment:
Not sure I follow, what's the key here? Abstracting sha256 to a helper func?

Reply:
yeah, as CC_MD5 will disappear soon, just future-proofing.